tests/functional/lib/helpers.js

```diff
@@ -708,6 +708,7 @@ define([
         return this.parent
           .then(openPage(endpoint, expectedHeader))
           .then(function () {
+            // > 1 because email must always be specified
             if (Object.keys(queryParams).length > 1) {
```

Comment:
Good question, we sometimes open the force_auth page w/o the email to test query parameter validation is working as expected. Maybe I should update the comment to say that.